dora/core/server/master/src/main/java/alluxio/master/job/CopyJob.java

```diff
@@ -633,28 +657,40 @@ private String getTextReport() {
       StringBuilder progress = new StringBuilder();
       progress.append(
           format("\tSettings:\tcheck-content: %s%n", mCheckContent));
-      progress.append(format("\tJob State: %s%s%n", mJobState,
-          mFailureReason == null
-              ? "" : format(
-                  " (%s: %s)",
-              mFailureReason.getClass().getName(),
-              mFailureReason.getMessage())));
+      Timestamp ts = new Timestamp(mStartTime);
+      Date date = ts;
```

Comment:
any reason we assign variable here?

Reply:
To get the correct format of date string.

Reply:
then you can just do new Date, right?